numba/tests/test_np_functions.py

```diff
@@ -124,14 +124,41 @@ def tril_m(m):
 def tril_m_k(m, k=0):
     return np.tril(m, k)
 
+def tril_indices_n(n):
+    return np.tril_indices(n)
+
+def tril_indices_n_k(n, k=0):
+    return np.tril_indices(n, k)
+
+def tril_indices_n_k_m(n, k=0, m=None):
+    return np.tril_indices(n, k, m)
+
```

Comment:
Maybe you ought to add an additional function here for `np.tril_indices(n, m=None)` that exercises the variant where `k` is omitted?

Reply:
Done